apps/web/src/studio/components/workflows/step-editor/WorkflowsStepEditorPage.tsx

```diff
@@ -13,9 +13,13 @@ export const WorkflowsStepEditorPage = () => {
   const [payload, setPayload] = useState({});
   const { templateId = '', stepId = '' } = useParams<{ templateId: string; stepId: string }>();
 
-  const { data: workflow, isLoading } = useQuery(['workflow', templateId], async () => {
-    return bridgeApi.getWorkflow(templateId);
-  });
+  const { data: workflow, isLoading } = useQuery(
+    ['workflow', templateId],
+    async () => {
+      return bridgeApi.getWorkflow(templateId);
+    },
+    { refetchOnWindowFocus: 'always' }
```

Comment:
To make code changes shown immediately in the local studio when user returns to the screen after changing something - no need to refresh